index.js

```diff
@@ -15,7 +15,7 @@ const getWorker = (file, content, options) => {
     const fallbackWorkerPath = options.fallback === false ? 'null' : workerPublicPath;
     return `require(${createInlineWorkerPath})(${JSON.stringify(content)}, ${fallbackWorkerPath})`;
   }
-  return `new Worker(${workerPublicPath})`;
+  return `new Worker(${workerPublicPath} + window.location.search)`;

```

Comment:
If necessary this must be implemented as an option instead

Reply:
I took a shot at that, let me know what you think. I haven't added documentation yet, will do after we agree on the implementation.